hooks/useEmailInfo.js

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from "@tanstack/react-query";
+import { doc, getDoc } from "firebase/firestore";
+import { db } from "~/firebaseConfig";
+
+export const useEmailInfo = (userId) => {
+  const {
+    data: userEmail,
+    error,
+    isLoading,
+  } = useQuery({
+    queryKey: userId ? ["user", userId] : null,
```

Comment:
I think it should be fine if the key just has null in it, no need for an if here

Reply:
@solderq35 suggested adding this if. Jeff, should I remove it?

Reply:
I'm starting to move away from conditional queryKey because it doesn't matter what it is, it just has to be different. this is the queryKey for attempts: `    queryKey: ["attempts", currentTeamId, { attemptId, userId, drillId }],` even though attemptId, userId, or drillId can be null

Reply:
yeah I guess you could reuse the syntax from useAttempts hook